remix/app/routes/_app.newsletter.update.$id.tsx

```diff
@@ -42,11 +44,29 @@ export async function action({ request, params }: ActionArgs) {
   });
 }
 
+export const loader = async ({ params }: ActionArgs) => {
+  const newsletter = await NewsletterRepository.findOne({
+    where: { id: params.id },
```

Comment:
I think we should also pass the `userId` into this query, right? 🤔

Reply:
Umm, that's my bad I missed there 🙏🏼

Reply:
Added here: [b4eddfe](https://github.com/nimblehq/js-framework-benchmark/pull/130/commits/b4eddfee4c7915ecdf0fca12edc7b25d45db97d8)

Reply:
It's better now! Just curious, why did you also change the method's name to `findFirst`? 👀

Reply:
Since I already have `findMany`, I think is better to name it `findFirst` too, I think is better to clarify the behavior of the method from the name too.